src/definitions/modules/modal.less

```diff
@@ -33,7 +33,12 @@
     flex: 0 0 auto;
     border-radius: @borderRadius;
     user-select: text;
-    will-change: top, left, margin, transform, opacity;
+    will-change:
```

Comment:
this this patching based on comma present ok?

Reply:
> this this patching based on comma present ok?

It's ok, although the simple liste for will-change would be sufficient in one line. But if your prefer it here, i would agree.

> is semicolon on the same like ok? (in other cases, it cannot be patched easily, also, when Prettier decised to wrap (usually based on maxLength), it also does not add a line break before the semicolon)

Moving the semicolon to the end is ok, although having it in a separate line as it is now more visualises the related block. But we had the same discussion for the JS files (var lists) as well 😉 and the argument of "i still have to change the previous line by adding a comma when i want to add another parameter" would be consistent and i agreed there to have the semicolon in the same line, so i would agree here too, if you think its better 🙂

Reply:
> semicolon placement

I want to stay as much as possible with the Prettier CS which is broadly accepted.

In JS we place semicolon at the same line too (for ex. for ternary), in declaration, we should actually switch to multiple `let`/`const`, so then there will be no semicolon on a separate line